sys-apps/shadow/shadow-4.4.ebuild

```diff
@@ -117,15 +131,14 @@ src_install() {
 	newins etc/login.defs login.defs
 	dosym ../usr/share/shadow/login.defs /etc/login.defs
 
+	set_login_opt CREATE_HOME yes
```

Comment:
Note: This is a behavioural difference. The upstream ebuild added it as well.
I'm not sure whether we particularly care whether this is set or not, but happy to remove it or leave it either way.